Price themes

Price47 opened the pull request:
Add actual good themes (even while being boxed into ridiculous standards)

Commit:
NF: Add themes

PFC and Space

(obviously update signature as well)

Commit:
-RF: Update sig

xtream1101 commented on src/server.py:
```diff
@@ -34,7 +34,8 @@
     },
     {"type": "section", "text": {"type": "mrkdwn", "text": ""}},
-    {"type": "context", "elements": [{"type": "mrkdwn", "text": "This slackbot was created by Eddy Hintze"}]}
+    {"type": "context", "elements": [{"type": "mrkdwn", "text": "This slackbot was created by Eddy Hintze and "
```
Nice try price, this is not going to fly

Price47 replied:
I think its important that people know

Commit:
NF: Chalkboard theme

Commit:
-RF: Allow an unbelievable injustice to occur

Commit:
-RF: And yet

Just when you think no greater injustice can occur, the beast of pride and arrogance rears it's ugly head to crush my soul just a tiny bit more
